pkg/controller/acmechallenges/scheduler/scheduler_test.go

```diff
@@ -41,7 +41,7 @@ func randomChallenge(rand int) *cmacme.Challenge {
 	}
 	return gen.Challenge("test-"+util.RandStringRunes(10),
 		gen.SetChallengeDNSName(util.RandStringRunes(rand)),
-		gen.SetChallengeType("http-01"))
+		gen.SetChallengeType(cmacme.ACMEChallengeTypeHTTP01))
```

Comment:
I decided to change the signature of SetChallengeType, to avoid this problem in the future.